massgen/orchestrator.py

```
@@ -13429,6 +13442,546 @@ async def _spawn_trace_analyzer_background(
+                    StructuredEventType.EVALUATION_CRITERIA_SET,
+                    criteria=evolved_payload,
+                    source=f"evolved_v{self._criteria_evolution_count}",
+                )
+            display = getattr(self.coordination_ui, "display", None) if self.coordination_ui else None
+            if display and hasattr(display, "set_evaluation_criteria"):
+                display.set_evaluation_criteria(
+                    evolved_payload,
+                    source=f"evolved_v{self._criteria_evolution_count}",
+                )
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)